hybris/egl/egl.c

```diff
@@ -425,14 +425,12 @@ __eglMustCastToProperFunctionPointerType eglGetProcAddress(const char *procname)
 			ret = _hybris_libgles1 ? dlsym(_hybris_libgles1, procname) : NULL;
 			break;
 		case 2:  // OpenGL ES 2.0 API
+		case 3:  // OpenGL ES 3.x API
```

Comment:
i know that on android libGLESv3 is usually the same as libGLESv2 but should we still load libGLESv3 in the glesv3 case just in case that changes?